src/_locales/pt_BR/messages.yml

```diff
@@ -167,27 +173,31 @@ editValueSave:
   message: Salvar
 extDescription:
   description: 'Description for this extension, will be displayed in web store'
-  message: Fornece suporte a scripts de usuários para navegadores.
+  message: Um gerenciador de script de usuário de código aberto que tem suporte à muitos navegadores
 extName:
   description: Name of this extension.
   message: Violentmonkey
 failureReasonBlacklisted:
   description: 'Shown for blacklisted URLs (in the extension icon tooltip, in the popup)'
-  message: ''
+  message: Configurações de Lista negra do Violentmonkey
 failureReasonNoninjectable:
   description: >-
     Shown for URLs that cannot be processed (same places as
     failureBlacklistedUrl)
-  message: ''
+  message: >-
+    Violentmonkey não pode executar scripts de usuários nesta página
+
+	(exemplos comuns: interface do navegador, uma extensão, bloqueada via políticas, uma motor de pesquisa 
```

Comment:
Sorry, this is still not incorrect. You can review your changes here under `Files changed` tab.
Maybe you should check your YAML with some online tools first.